include/aspect/postprocess/stokes_residual.h

```diff
@@ -0,0 +1,121 @@
+/*
+  Copyright (C) 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__postprocess_depth_average_h
+#define __aspect__postprocess_depth_average_h
+
+#include <aspect/postprocess/interface.h>
+#include <aspect/simulator_access.h>
+
+#include <deal.II/base/data_out_base.h>
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+
+    /**
+     * A postprocessor that outputs Stokes residuals
+     *
+     * @ingroup Postprocessing
+     */
+    template <int dim>
+    class StokesResidual : public Interface<dim>, public ::aspect::SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Constructor.
+         */
+        StokesResidual ();
+
+        /**
+         * Generate graphical output from the current solution.
```

Comment:
it's probably not graphical. I imagine you probably copied this from elsewhere, though...